.circleci/config.yml

```diff
@@ -302,3 +302,32 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+
+      # vsphere
+      - architect/push-to-app-collection:
```

Comment:
We will refactor this repo soon - so is there a specific reason you need this? Otherwise you could wait a bit and I will work on straightening this out.

Reply:
nope that's fine with me, i'm just aligning vsphere with the openstack collection - no urgency here.